Civi/Api4/Generic/AbstractAction.php

```diff
@@ -44,6 +46,16 @@ abstract class AbstractAction implements \ArrayAccess {
    */
   protected $version = 4;
 
+  /**
+   * Preferred Language (optional).
+   *
+   * If set then listeners such as the Translation subsystem may alter
+   * the output.
+   *
+   * @var string
```

Comment:
If you grep for `@option`, then there are a couple examples like:

```
Civi/Api4/Generic/AbstractSaveAction.php:   * @optionsCallback getMatchFields
Civi/Api4/Generic/ExportAction.php:   * @options never,always,unmodified
Civi/Api4/Generic/ExportAction.php:   * @options never,always,unused
Civi/Api4/Action/WorkflowMessage/GetTemplateFields.php:   * @options metadata,example
Civi/Api4/Action/WorkflowMessage/Render.php:   * @options error,warning,info
```

So this could probably use an `@optionsCallback`?